src/quo2/components/graph/wallet_graph/style.cljs

```diff
@@ -0,0 +1,18 @@
+(ns quo2.components.graph.wallet-graph.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def gradient-background
+  {:height          294
+   :justify-content :flex-end})
+
+(def x-axis-label-text-style
+  {:margin-bottom -3
+   :padding-top   -10
+   :height        0})
+
```

Comment:
This styling looks suspicious. Why negative values, and height is zero? Can't it be done in another way?

Reply:
It's a tweak to remove bottom bar from the chart. Not ideal, but didn't find a better solution. I'll add a comment so it is easier to understand.